---
layout: bio
categories: faculty
title: Nevan Krogan
position: Professor
email_prefix: nevan.krogan
email_suffix: ucsf, edu
phone: 415 476-2980
fax:
website: https://kroganlab.ucsf.edu/
address: 1700 4th Street, MC 2530</br>Byers Hall, Room 308D</br>San Francisco, CA 94158-2530
info: Executive Assistant</br>Lauren Weiser</br>415 734-4809 </br><span class="e">lauren.weiser / gladstone.ucsf, edu</span></br>


image: krogan.jpg
---

My research group focuses on fundamental biological mechanisms, because cures to many diseases have been revealed by unexpected discoveries in the basic sciences. Research in the Krogan lab is disease-agnostic, and makes use of a wide range of systems biology approaches. Complementing technologies that allow the unbiased study of protein-protein interactions, protein modifications, and the effects of combinatorial gene deletions help us inform targeted mechanistic and structural studies, paving the way towards drug development. We strongly believe that impactful research is accomplished when diverse groups of scientists work together, and therefore we are working in close collaboration with experts from different disciplines on all of our projects.
The disease areas we are currently studying include, but are not limited to: Infectious Diseases, Cancer, Psychiatric Disorders, Tauopathies and Heart Disease.

One of the new approaches developed in collaboration with experts across the UC system and beyond is reflected in a variety of ‘**Cell Mapping Initiatives**’, which we apply to many biological or biomedical problems, including infectious diseases ([HPMI](http://hpmi.ucsf.edu/)), cancer ([CCMI](http://www.ccmi.org/)) as well as neurological and psychiatric disorders ([PCMI](http://pcmi.ucsf.edu/)). In these projects, we create cell maps by comprehensively identifying the protein complexes that work together to ensure healthy cell physiology.
For this, we rely heavily on a wide variety of **proteomics techniques** including: unbiased global proteome and PTM characterization with label-free quantitation, large-scale quantitative affinity purification mass spectrometry (AP-MS) to define protein networks, targeted quantitation via selected reaction monitoring (SRM), structural studies using cross-linking mass spectrometry (XL-MS), as well as proximity labeling mass spectrometry (APEX-MS) to study short-lived interactions (such as signaling networks) and resolve the cellular location of interactions using spatial references. A primary interest in the lab is combining several of these complementary MS-based approaches in a systems biology fashion to develop a comprehensive picture of the biological system of interest. We perform all our MS-based experiments as part of the Thermo Fisher Scientific Proteomics Facility for Disease Target Discovery. **Functional genomics approaches** further help us identify and study the importance of specific molecular networks, and place protein complexes into pathways. Our expertise spans pairwise perturbations of genes across different organisms (Genetic Interaction Mapping), as well as more targeted approaches to study the mechanism underlying a specific phenotype, using CRISPR/Cas9 gene editing. We are also developing **bioinformatics tools** for interpreting the highly complex data resulting from proteomics and functional genomics platforms in order to maximize the extraction of functionally relevant biological knowledge. Combining all this information, we study how disease mutations or infections adversely change protein networks and, ultimately, how drugs or other treatments can overcome these changes.
