---
layout: post
category: publication
published: true
title: "Dynamic translation regulation in Caulobacter cell cycle control."
date: 2016-10-28 12:00:00
pmid: 27791168
authors: "Schrader JM, Li GW, Childers WS, Perez AM, Weissman JS, Shapiro L, McAdams HH"
firstauthor: "Schrader JM"
journalname: "Proc. Natl. Acad. Sci. U.S.A."
journalvolume: 
journalissue: 
journalpages: 
---

Progression of the Caulobacter cell cycle requires temporal and spatial control of gene expression, culminating in an asymmetric cell division yielding distinct daughter cells. To explore the contribution of translational control, RNA-seq and ribosome profiling were used to assay global transcription and translation levels of individual genes at six times over the cell cycle. Translational efficiency (TE) was used as a metric for the relative rate of protein production from each mRNA. TE profiles with similar cell cycle patterns were found across multiple clusters of genes, including those in operons or in subsets of operons. Collections of genes associated with central cell cycle functional modules (e.g., biosynthesis of stalk, flagellum, or chemotaxis machinery) have consistent but different TE temporal patterns, independent of their operon organization. Differential translation of operon-encoded genes facilitates precise cell cycle-timing for the dynamic assembly of multiprotein complexes, such as the flagellum and the stalk and the correct positioning of regulatory proteins to specific cell poles. The cell cycle-regulatory pathways that produce specific temporal TE patterns are separate from-but highly coordinated with-the transcriptional cell cycle circuitry, suggesting that the scheduling of translational regulation is organized by the same cyclical regulatory circuit that directs the transcriptional control of the Caulobacter cell cycle.

